frontend/apps/reader/modules/readersearch.lua

```diff
@@ -28,6 +28,14 @@ function ReaderSearch:addToMainMenu(menu_items)
             self:onShowFulltextSearchInput()
         end,
     }
+    if self.ui.document.provider == "crengine" then
+        menu_items.regex_search = {
+            text = _("Regular expression search"),
```

Comment:
```
Fulltext search
Regular expression search
```
I guess I'm ok with that.
Just a question for our english scholars: "fulltext" means it's a search in the full book ? Or just that it's a plain search for the full pattern?
Is the regular expression search is also a fulltext search? May be `Fulltext search (regex)` if we keep 2 menu items.

Reply:
It's more programmerish than English. That being said, just `search` would be a bit cryptic, so you do need it to be (full)text search or search in text.

I don't think the full is necessarily bad though, because it indicates that the full text of the book will be searched, rather than just the current page. But I'm not sure if that potential confusion actually occurs IRL.